modules.py

```diff
@@ -143,10 +109,9 @@ def P_rated(x_in, x_name, p_in: dict):
 def carrying_capacity_print(x_in, x_name, p_in: dict):
```

Comment:
i've noticed this quite a few times throughout this code, but i've noticed several functions don't have return statements. i'm not a python-pro, so i don't know if there's a specific reason for this, but i thought no return statement = nothing is returned from the function ?

Reply:
they've been used to print some values and don't return any results.